tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -157,4 +157,37 @@ Feature: create auth-app token
     And the content in the response should include the following content:
       """
       error parsing expiry. Use e.g. 30m or 72h
-      """
\ No newline at end of file
+      """
+
+  @env-config
+  Scenario: admin creates auth-app token with user-id
+    Given the config "AUTH_APP_ENABLE_IMPERSONATION" has been set to "true"
```

Comment:
so we need  this env "AUTH_APP_ENABLE_IMPERSONATION" when creating token for another user?

Reply:
I was confused because of you another PR, but gets cleared up by your reply